packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs-table-actions.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable camelcase -- ...*/
+import { Link, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useRejectTaskMutation } from '@/api/services/worker/reject-task';
+import { TableButton } from '@/components/ui/table-button';
+import { RejectButton } from '@/pages/worker/jobs/components/reject-button';
+import type { MyJob } from '@/api/services/worker/my-jobs-data';
+
+interface MyJobsTableRejectActionProps {
+  page: MyJob;
```

Comment:
```suggestion
  job: MyJob;
```

`page` is very confusing here

Reply:
Done @dnechay